### PR TITLE
D8CORE-2268: adjusting the spacing for media files in the wysiwyg and fixing the centering for the pdf.

### PR DESCRIPTION
**Pull request by jenbreese**
# READY FOR REVIEW

# Summary
- Fixing the alignment of media that are files added to the WYSIWYG

# Review By (Date)
- 12/3

# Urgency
- Med

# Steps to Test

1. Pull in change.
2. Open a page and edit text. 
3. Embed a media file. Use the "Edit Media" to align one left, one right, one centered and one with none. 
4. Verify the items are aligned with padding on the approprieate sides.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- D8CORE-2268
- @cjwest 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by jenbreese** [comment]
@cjwest All ready for you. The code indentation is only one and not normally a problem.

**Comment by cjwest** [comment]
@jenbreese,
I don't feel comfortable approving this with the codeclimate issue. It appears that you need only add a space on line 19.

File: src/scss/components/wysiwyg/_media.scss [hunk 1]
```diff
@@ -6,5 +6,27 @@
...
+    .media {
+      &.file {
+        text-align: center;
+      }
+    }
+  }
+
+  .align-left {
+    .file {
+       &.media {
```
Comment (sherakama) [inline src/scss/components/wysiwyg/_media.scss:19]:
```suggestion
      &.media {
```
Help a brother out.

Comment (jenbreese) [reply src/scss/components/wysiwyg/_media.scss:19]: Dang it. Of course I'll fix.

Comment (jenbreese) [reply src/scss/components/wysiwyg/_media.scss]: I see what happened. I misread the indention as too many levels not that I had an additional space. I'm sorry @cjwest and @sherakama! :)

**Comment by jenbreese** [comment]
hi @cjwest , I got my mistake on the code climate fixed. Can you take a look and merge if ok?

**Comment by cjwest** [comment]
@jenbreese,
Pulled it in, tested, looks good!

**Review by cjwest — APPROVED** [review]

**Comment by sherakama** [comment]
Thanks ya'll. You're doing great work.

**Pull request merged**